server/app/src/main/java/com/ams/app/service/SubjectService.java

```diff
@@ -0,0 +1,79 @@
+package com.ams.app.service;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+import com.ams.app.model.Subject;
+import com.ams.app.repository.SubjectRepository;
+
+@Service
+public class SubjectService {
+    
+    @Autowired
+    private SubjectRepository Srepo;
```

Comment:
Use properName like studentRepository for this. Don't use shortcuts

Reply:
Renamed to studentRepository